### PR TITLE
Dockerization

### PR DESCRIPTION
**Pull request by djodjo3**
* Added Dockerfile to locally build the docker image
* Added build and run instruction to the README

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+FROM alpine AS builder1
+
+ARG CROWDIN_TOKEN
+ENV ENV_CROWDIN_TOKEN=$CROWDIN_TOKEN
```
Comment (Quetzacoalt91) [inline Dockerfile]: Is this line necessary? We won't need the value later, as we only have a temporary stage.

Comment (djodjo3) [reply Dockerfile]: Good catch, we don't need to propagate this 👍 
Will correct it right now.

Comment (djodjo3) [reply Dockerfile]: And that should do it !

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @djodjo3

**Pull request merged**